administrator/language/en-GB/joomla.ini

```
@@ -86,6 +86,7 @@ JFEATURED="Featured"
 JFEATURED_ASC="Featured ascending"
 JFEATURED_DESC="Featured descending"
 JHELP="Help"
+JINLINEHELP="Show/Hide Inline Help"
```

```suggestion
JINLINEHELP="Toggle Inline Help"
```
Please consider to be consistent with `Toggle Menu` and `Toggle Editor`.

---

I agree with that change. I'll find other instances of that and change that too.